### PR TITLE
stylo: Support remaning longhands

### PR DESCRIPTION
**Pull request by Manishearth**
r=xidorn https://bugzilla.mozilla.org/show_bug.cgi?id=1367275

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17058)

### CODE REVIEW

**Comment by Manishearth** [comment]
(will try to land tomorrow)

File: components/style/properties/longhand/font.mako.rs [hunk 1]
```diff
@@ -1785,125 +1786,58 @@ ${helpers.single_keyword_system("font-variant-position",
...
-
-                match self.value {
-                    1 => Ok(()),
-                    0 => dest.write_str(" off"),
-                    x => write!(dest, " {}", x)
-                }
-            }
-        }
+<%
+# This is ridiculous
```
Comment (nox) [inline components/style/properties/longhand/font.mako.rs]: Remove that.

Comment (Manishearth) [inline components/style/properties/longhand/font.mako.rs]: Aw. Fine.

**Comment by Manishearth** [comment]
@bors-servo r=emilio

    infra exception

**Pull request merged**